src/vs/workbench/electron-browser/workbench.main.ts

```diff
@@ -78,6 +78,7 @@ import 'vs/workbench/parts/trust/electron-browser/unsupportedWorkspaceSettings.c
 import 'vs/workbench/parts/relauncher/electron-browser/relauncher.contribution';
 
 import 'vs/workbench/parts/tasks/electron-browser/task.contribution';
+import 'vs/workbench/parts/tasks/common/taskActions';
```

Comment:
I think this should be in task.contribution.ts, which pulls in other task-related files.